xwiki-platform-core/xwiki-platform-release/xwiki-platform-release-ui/pom.xml

```diff
@@ -66,9 +66,8 @@
     <!-- We need the livetable for the Release Home page that lists all Releases -->
     <dependency>
       <groupId>org.xwiki.platform</groupId>
-      <artifactId>xwiki-platform-livetable-ui</artifactId>
+      <artifactId>xwiki-platform-livedata-macro</artifactId>
```

Comment:
You need to keep the ``livetable-ui`` (for the default results page) and add the dependency to the live table source. This module basically uses the live data **macro** with the live table **source** with the default live table **results page**. So it depends on all 3.